packages/@webex/plugin-meetings/src/breakouts/collection.ts

```diff
@@ -0,0 +1,19 @@
+/*!
+ * Copyright (c) 2015-2023 Cisco Systems, Inc. See LICENSE file.
+ */
+
+import AmpCollection from 'ampersand-collection';
+
+import {MEETINGS} from '../constants';
+
+import Breakout from './breakout';
+
+const BreakoutCollection = AmpCollection.extend({
```

Comment:
What is this `AmpCollection` dependency, and why does it help us here?

Reply:
We use ampersand state in the SDK. Ampersand state has the concept of collections, groups of models. The collections offer helpful methods for searching, sorting models as well as event propagation. From what I can tell looking at the rest of the SDK, in places where authors didn't use ampersand collections, they had to resort to writing a lot of extra custom cost to do things like a get a model by its ID or to listen to an event within a model in a collection. There's no need for all that extra code, since the framework we already chose does this for us.

Reply:
Thanks for the thorough explanation! Makes sense!

Reply:
we will need the same for members at some point